tools/Performance/DynamoPerformanceTests/DynamoViewModelPerformanceTestBase.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using BenchmarkDotNet.Attributes;
+using DynamoCoreWpfTests;
+
+namespace DynamoPerformanceTests
+{
+
+    public class DynamoViewModelPerformanceTestBase : NodeViewTests
```

Comment:
this name is wrong now since the base class creates a view - not a viewModel

Reply:
Sure, make sense. Will update